teemaderegister-react/src/components/addNewUserForm.jsx

```diff
@@ -0,0 +1,108 @@
+import React, {useEffect, useState} from 'react'
+// eslint-disable-next-line standard/object-curly-even-spacing
+import {Button, Form, Input, notification, Select} from 'antd'
+import {adminAddNewUser} from '../actions/AdminActions'
+import {connect} from 'react-redux'
+import { setDocTitle } from '../utils/Helpers'
+
+const {Option} = Select
+
+const AddNewUserForm = (props) => {
+  const [form] = Form.useForm()
+  const [isLoading, setIsLoading] = useState(false)
+  const roles = [
+    'admin',
+    'curriculum-manager',
+    'head-of-study',
+    'student',
+    'study-assistant',
+    'supervisor'
+  ]
+  setDocTitle('Add new user')
```

Comment:
is it called once or there can be some sideeffects? Would do it once when component loads (useEffect can be used)